core/src/main/java/org/mskcc/cbio/oncokb/util/MainUtils.java

```diff
@@ -157,6 +157,17 @@ public static IndicatorQueryMutationEffect findHighestMutationEffectByEvidence(S
         return indicatorQueryMutationEffect;
     }
 
+    public static IndicatorQueryMutationEffect setToAlternativeAlleleMutationEffect(IndicatorQueryMutationEffect indicatorQueryMutationEffect) {
+        if (indicatorQueryMutationEffect != null && indicatorQueryMutationEffect.getMutationEffect() != null) {
+            MutationEffect mutationEffect = indicatorQueryMutationEffect.getMutationEffect();
+            MutationEffect likeME = MutationEffect.getByName("Likely " + mutationEffect.getMutationEffect().replaceAll("(?i)likely", "").trim());
+            if (likeME == null || likeME.equals(MutationEffect.LIKELY_NEUTRAL))
```

Comment:
If I understand correctly, likeME == null means the mutationEffect is inconclusive? If that is the case, could you please add some comment there?

Reply:
Sure